### ISSUE TITLE
Hightlighting doesn't work with nested template string and with css helper. 

### ISSUE THREAD

**Issue by Avivhdr** [styled-components/vscode-styled-components#261]
1. Doesn't highlight code if I use a tagged template inside the template string (see middle image) (#254 ).
2. The above problem (1) can be solved by using css helper before the tagged template.
3. The above solution (2) cause highlight not to work further down (see right screenshot)

(The solution does work well on codesandbox but not in vscode - [see here](https://codesandbox.io/s/styled-components-oqll7?file=/test.js))

**Screenshot**
![image](https://user-images.githubusercontent.com/23265867/101985036-a7e43580-3c85-11eb-910f-57bce7a04ca1.png)

Code for left & middle screenshots:
```
const box1 = styled.div`
  display: grid;
  grid-template-columns: repeat(9, 1fr);
  grid-column: 6/11;
  color: ${myColor};
  color: red;
  ${
    setMediaQuery('big')`
      display: grid;
      color: ${myColor};
      color: red;
      grid-column: 6/11;
      grid-template-columns: repeat(9, 1fr);
    `
  }

  ${
    setMediaQuery('medium')`
      display: grid;
      grid-template-columns: repeat(9, 1fr);
      grid-column: 6/11;
      color: red;
    `
  }
`;

const box2 = styled.div`
  display: grid;
  grid-template-columns: repeat(9, 1fr);
  grid-column: 6/11;
  color: ${myColor};
`;

export default {
  box1,
  box2,
};
```

Code for the right screenshot:
```
import styled, { css } from 'styled-components';

const myColor = '#333';

export const setMediaQuery = (breakpoint) => {
  switch (breakpoint) {
    case 'small':
      return (style) => `@media (max-width: 800px) { ${style} }`;
    case 'medium':
      return (style) => `@media (min-width: 801px) (max-width: 1200) { ${style} }`;

    default:
      return (style) => `@media (min-width: 1201px) { ${style} }`;
  }
};

const box1 = styled.div`
  display: grid;
  grid-template-columns: repeat(9, 1fr);
  grid-column: 6/11;
  color: ${myColor};
  color: red;
  ${
    setMediaQuery('big')(css`
      display: grid;
      color: ${myColor};
      color: red;
      grid-column: 6/11;
      grid-template-columns: repeat(9, 1fr);
    `)
  }

  ${
    setMediaQuery('medium')(css`
      display: grid;
      grid-template-columns: repeat(9, 1fr);
      grid-column: 6/11;
      color: red;
    `)
  }
`;

const box2 = styled.div`
  display: grid;
  grid-template-columns: repeat(9, 1fr);
  grid-column: 6/11;
  color: ${myColor};
`;

export default {
  box1,
  box2,
};
```

- OS: Win 10
- VSCode Version: 1.52.0
- Extension Version 1.4.1

Thanks!

**Comment by jasonwilliams** [comment]
@Avivhdr are you interested in working on this problem? We are low on contributors right now so I don’t know when it will get looked at.

https://github.com/styled-components/vscode-styled-components/blob/master/CONTRIBUTING.md

**Comment by stg101** [comment]
I can work on it if @Avivhdr hasn't started yet

**Comment by jasonwilliams** [comment]
Thanks @stg101

**Comment by Avivhdr** [comment]
No, I haven't.
I forked it but I think it is above my current skill set.

**Comment by Avivhdr** [comment]
Thanks @stg101

**Comment by fadhilx** [comment]
is there any temporary solution for this?

**Comment by Avivhdr** [comment]
I Have the installed version 1.5, the version with the fix, but it didn't change the situation for me.
In this screenshot you can see (like in my initial screenshots that were attached to this issue) that the CSS inside the template function is not being taken care of:
* A reminder: In this [sandbox](https://codesandbox.io/s/styled-components-oqll7?file=/test.js) works properly, and did so even before version 1.5.
![image](https://user-images.githubusercontent.com/23265867/109933178-69ca4100-7ccb-11eb-9449-8e0c39b9e057.png)

**Comment by ndeviant** [comment]
Doesn't seems fixed for me as well :C
As you can see, there is 3 variants of writing styles, and it works only for the last option.

<img width="573" alt="Screenshot 2021-08-12 at 00 08 53" src="https://user-images.githubusercontent.com/25330222/129104408-4574841d-48c2-4b6c-aa3e-36fffc00c525.png">

**Comment by Avivhdr** [comment]
Hi @ndeviant,

See https://github.com/styled-components/vscode-styled-components/issues/304 or [this codepen](https://codesandbox.io/s/styled-components-oqll7?file=/test.js:28-77).

**TLDR:**
You can pass your styles to the built-in css helper:

```
import styled, { css } from "styled-components";

const TextSmall = styled(Typography)(css`
   font-size: 0.75 rem;
   margin-bottom: 12pc;
`)
```

**Comment by lodania** [comment]
instale "vscode-styled-components" desde las extensiones.
